docs/work-flow.md

```diff
@@ -0,0 +1,176 @@
+# DC/OS Development Workflow
+
+
+        "... there is absolutely nothing "obvious" about source-control systems, or workflows before you already know them"
+
+Mesosphere DC/OS has two primary repositories.
+
+1. Open Source DC/OS (https://github.com/dcos/dcos)
+2. Closed Source Mesosphere Enterprise DC/OS (https://github.com/mesosphere/dcos-enterprise)
+
+In our design, the closed source Mesosphere Enterprise DC/OS depends on the Open Source DC/OS, [builds on top of the artifact](https://github.com/mesosphere/dcos-enterprise/blob/master/util/build.py) from the Open source DC/OS.
```

Comment:
❓  "In our design, the closed source" > "Closed Source".

"Open source DC/OS" > "Open Source DC/OS".